Makefile

```diff
@@ -67,7 +67,9 @@ test: clean lint
 
 .PHONY: license
 license: clean tools
-	$(GO_LICENSER) -d -exclude=protocol/gen-codes -licensor='Apache Software Foundation (ASF)' .
+	$(GO_LICENSER) -d  -exclude=plugins/queue/mmap \
```

Comment:
You should not exclude the whole mmap folder. I believe not all of them are from a 3rd party. Please use the explicit file name list.

Reply:
The exclude feature of the licensor is only working in a dir.  When refactoring the file to a single dir, a serious problem would be occurring, that is circle reference.

Reply:
Why folder matters for source codes? Please be clear. You are risking the project missing the license header in the source release.

Reply:
for example, there are 2 influenced files, which are queue.go and queue_opreation.go. If refactoring the queue_opreation.go to bigqueue/queue_opreation.go, circle reference would be occurringing.  The ` queue.go` in mmap package imports bigqueue package, some operations in bigqueue package are also dependent on the map package.

Reply:
This is the typle case of decoupling the codes. I think there are enough design patterns to fix this.
Anyway, if tool supports file level exclusion, should be better.

Reply:
I will try my best to solve this problem